docs/upgrade/automatic.md

```diff
@@ -249,6 +249,91 @@ In production environments, upgrading clusters via the Harvester UI is recommend
 
 :::
 
+## Customize Node Upgrade
+
+_Available as of v1.7.0_
+
+Harvester Upgrade consists of several phases. The node upgrade phase is when Harvester upgrades each node's RKE2 and operating system sequentially and autonomously. You can explicitly specify which nodes should not perform such automatic node upgrades and can only continue doing so upon your consent. During the pause period, you can perform necessary maintenance tasks on the node. After everything is settled, you can instruct Harvester to proceed with the node upgrade for the paused node.
+
+### How to Pause a Node from Node Upgrade
+
+Check out the `nodeUpgradeOption` option under the `upgrade-config` setting. You can toggle the `mode` field from `auto` to `manual`, enabling pause for every node in the cluster during the node upgrade phase. You can further specify which nodes should be paused, and the remaining ones will still undergo the node upgrade autonomously. For more details about the relevant options, see the [`upgrade-config`](advanced/settings.md#upgrade-config) setting page.
+
+:::info
+
+Harvester applies the pause-relevant configuration during upgrade initialization. After that, any further changes to the fields under the `nodeUpgradeOption` option will not affect the current upgrade and can only take effect upon the next upgrade.
```

Comment:
add to here or other places, mention that when the upgrade is ongoing, as long as a node is not upgraded yet, it can be freely added or removed to/from the `upgrade` object's annotation;  the upgrade makes decision upon the `upgrade` object